src/components/Field/Field.tsx

```diff
@@ -70,14 +74,26 @@ export class Field extends React.PureComponent<FieldProps> {
     return (
       <div className={classes}>
         {label ? <Header sub>{label}</Header> : null}
-        <Input
-          value={value}
-          type={isAddress ? 'text' : type}
-          icon={icon}
-          loading={loading && !isAddress}
-          disabled={disabled}
-          {...rest}
-        />
+        {type === DATE_TYPE ? (
+          <SemanticDatepicker
+            value={value}
+            icon={icon ? icon : void 0}
+            loading={loading && !isAddress}
+            disabled={disabled}
+            format="DD-MM-YYYY"
+            showOutsideDays
+            className="datePickerWidth"
```

Comment:
This component doesn't seem to be receiving all the props as it's missing the `{...rest}`prop.